### PR TITLE
Fix docblocks for Builder class

### PR DESCRIPTION
**Pull request by bobbybouwmann**

### CODE REVIEW

File: src/Lavary/Menu/Builder.php [hunk 1]
```diff
@@ -602,24 +605,23 @@ protected static function attributeElement($key, $value)
...
-	 *
-	 * @return string
-	 */
+    /**
+     * Merge item's attributes with a static string of attributes
+     *
+     * @param null $new
+     * @param array $old
+     * @return string
+     */
```
Comment (dustingraham) [inline src/Lavary/Menu/Builder.php]: Looks like you converted the tabs to spaces here.

File: src/Lavary/Menu/Builder.php [hunk 2]
```diff
@@ -39,8 +39,8 @@ class Builder {
 	
 	/**
 	 * Initializing the menu manager
-	 *
-	 * @return void
+     *
+     * @return void
```
Comment (dustingraham) [inline src/Lavary/Menu/Builder.php]: Another tabs to spaces conversion.

**Comment by dustingraham** [comment]
Going to pull this in now before I clean up some other stuff in this file, so I'll be fixing it anyways, thanks.

**Comment by bobbybouwmann** [comment]
I didn't had time yet! Anyway, thank you for merging ;)

**Pull request merged**